package.json

```diff
@@ -35,8 +35,8 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",
-    "test": "yarn run test:type && yarn run test:dumb",
-    "test:unit": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
+    "test": "yarn run test:type && yarn run test:unit",
```

Comment:
@kavimaluskam Do you think I should put `npm` here? Is `yarn` here ok?

Reply:
I personally think `yarn` is ok. 
Randomly searched some approach on internet as well:
<https://stackoverflow.com/questions/51768743/how-to-detect-that-the-script-is-running-with-npm-or-yarn>

Reply:
Cool, but let's leave it as it is now. I prefer not install a lib to support cross platform for only this usage.
Using `yarn` looks good to me.